apps/demo-game/src/pages/admin/games.tsx

```diff
@@ -94,14 +94,29 @@ function Games() {
         )}
       </Formik>
 
-      <div className="mt-4 grid grid-cols-10 gap-1">
-        {data.games.map((game) => (
-          <div className="border p-2" key={game?.id}>
-            <Link href={`/admin/games/${game?.id}`}>
-              {game?.name} {game?.activePeriodIx}
-            </Link>
-          </div>
-        ))}
+      <div className="mt-4 flex flex-col gap-1">
+        {[...data.games]
+          .sort((gameA, gameB) => Number(gameB.id) - Number(gameA.id))
+          .map((game, index, array) => {
+            return (
+              <div className="flex w-96 border p-2" key={game?.id}>
```

Comment:
as the entire thing is a clickable interaction and leads to the game cockpit for the game master, make it a button so it has hover states and is more accessible and can be tabbed through